inst/odin/model.R

```diff
@@ -56,10 +56,14 @@ Np[]    <- sum(N[i, ]) * p[i]
 foi_LH[] <- prop_C[i] * Np[i] / sum(Np[])
 lambda[] <- p[i] * beta * (epsilon * prop_C[i] + (1 - epsilon) * sum(foi_LH[]))
 
-
 n_UI[, ]     <- lambda[i] * (1 - vea[j]) * U[i, j]
 n_AT[, ]     <- eta[i] * A[i, j]
-n_AU[, ]     <- (ved[j] * (mu - nu) + nu) * A[i, j]
+n_AU[, ]     <- if (mu == 0 || nu == 0) (nu / (1 - ved[j]) *
+                                           A[i, j]) else 1 / (ved[j] *
+                                                                ((1 / mu) -
+                                                                   (1 / nu)) +
+                                                                (1 / nu)) *
+  A[i, j]
```

Comment:
```suggestion
 n_AU[, ] <- if (mu == 0 && nu == 0) 0 else
  nu * mu / (ved[j] * nu + (1 - ved[j]) * mu) * A[i, j]
```
Suggest this formulation is a little easier to read and more intuitive to understand. In the PR version we revert to the old formulation of ve_d effect when mu == 0, which is essentially a different model. Given that we use this only for testing that seems at cross-purposes.